### DISCUSSION TITLE
Irelevant link from Skype

### DISCUSSION THREAD

**Discussion by msecara** [MicrosoftDocs/office-developer-client-docs#657] [Q&A]
Skype for Business &gt; Settings &gt; Video device &gt; Learn More points to this page, which is totally irrelevant for the Video Devices I wanted to setup in Skype

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e88d1d56-b2a7-0e79-74d6-60f84591c103
* Version Independent ID: a02dcc8f-46eb-8f94-a781-a70d1a6b6c82
* Content: [Office client development](https://docs.microsoft.com/en-us/office/client-developer/office-client-development#feedback)
* Content Source: [docs/office-client-development.md](https://github.com/MicrosoftDocs/office-developer-client-docs/blob/master/docs/office-client-development.md)
* Product: **office365**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by ghost** [comment]
@msecara can you please provide a link to the page that has the "bad" link, and if possible, let me know what you expect to find? Thanks...

**Comment by msecara** [comment]
Hello Linda,

This is how to reproduce:
Prerequisites: Skype for Business with working Webcam
Steps:
1. Open Skype for Business
2. Navigate to Options > Video Device
- The user sees the preview of the webcam
- The user has a link "Learn more"
3. Click the "Learn More" button

Expected:
The user should be redirected to a page when he can see webcam related
settings and configuration
Observed:
The user is redirected to page
https://docs.microsoft.com/en-us/office/client-developer/office-client-development,
which is about Office client development, but nothing related to the Webcam
settings, not even related to Skype at all.

Hope this helps.

Thank you,
Mihail

**Comment by ghost** [comment]
@msecara Thanks. I tried your steps, and when I go there, I get this result after clicking Learn More, which seems relevant: https://support.office.com/en-us/article/top-help-topics-22e8b9d2-f244-49e2-8715-d2807b8602f8?NS=SKYPEFB_ONLINE&Version=90&SysLcid=1033&UiLcid=1033&AppVer=MSO900&HelpId=185479&ui=en-US&rs=en-US&ad=US

**Comment by ghost** [comment]
@msecara Here's the link that page points to: https://support.office.com/en-us/skype-for-business

**Comment by sammie-p** [comment]
I am also directed here from the Skype for Business app. On the Add Contacts page, Skype Directory, there is a note: "Search for Skype contacts by name, Skype Name, email address, phone number, and location", followed by a blue question mark symbol. Click on the blue question mark, and I am directed here: https://officeredir.microsoft.com/r/rlidOfficeClientDeveloperHelp?ver=16&app=lync.exe&clid=1033&lidhelp=0409&liduser=0C09&lidui=0409

**Comment by ghost** [comment]
@sammie-p Thanks for the feedback. This is an issue in the product, not in the documentation. I'll see if I can track down someone who can fix this.
